Sources/App/Commands/Analyzer.swift

```diff
@@ -161,6 +201,11 @@ func getManifest(package: Package, version: Version) -> Result<Manifest, Error>
             throw AppError.invalidRevision(version.id, version.tagName)
         }
         try Current.shell.run(command: .gitCheckout(branch: revision), at: cacheDir)
+        guard Current.fileManager.fileExists(atPath: cacheDir + "/Package.swift") else {
+            // It's important to check for Package.swift - otherwise `dump-package` will go
+            // up the tree through parent directories to find one
+            throw AppError.invalidRevision(version.id, "no Package.swift")
+        }
```

Comment:
This whole `analyze` function needs some refactoring and cleanup but I don’t want to do that before we’ve fixed the `wait` workaround.